ovs_performance.py

```diff
@@ -571,7 +574,7 @@ def test_p2p(nr_of_flows, packet_sizes):
         if config.warm_up:
             lprint("  * Doing flow table warm-up...")
             tester.start_traffic(config.tester_interface)
-            warm_up_verify(nr_of_flows, DEFAULT_WARM_UP_TIMEOUT)
+            warm_up_verify(nr_of_flows, config.warm_up_timeout)
```

Comment:
This is an example why your change does not work. If it would fail here, there is no VM running.